the-blue-alliance-ios/Assets.xcassets/ic_arrow.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
We usually use material design icons, since it's what Android uses, and they're all fairly standard https://material.io/tools/icons

The chevron right or arrow forward looks like it'd work.